NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp

```diff
@@ -1167,7 +1183,7 @@ uint32 CGameClientShell::OnEngineInitialized(RMode *pMode, LTGUID *pAppGuid)
 		
 		// Use the settings from the current profile to determine what the game will be...
 
-		CUserProfile *pProfile = g_pProfileMgr->GetCurrentProfile( );
+		pProfile = g_pProfileMgr->GetCurrentProfile( );
```

Comment:
does pProfile change between 1123 and here?
if not how about dropping this line?